src/screens/ENSAssignRecordsSheet.js

```diff
@@ -16,33 +28,60 @@ export default function ENSAssignRecordsSheet() {
     navigate(Routes.ENS_SEARCH_SHEET);
   }, [navigate]);
 
+  const avatarColor =
+    usePersistentDominantColorFromImage('TODO').result || colors.purple;
```

Comment:
not sure if this is temporary, but if we are skipping avatars for phase 1, we probs want to remove this for performance reasons